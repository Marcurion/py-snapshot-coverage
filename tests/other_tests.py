import pytest
from syrupy import snapshot



@pytest.mark.parametrize(
    "a,b",(
    (1,2),
    (3,1),
    (4,2),
    (5,3),)
)
def test_smaller_than_five(snapshot, a, b):
    assert ((a+b)<5) == snapshot
