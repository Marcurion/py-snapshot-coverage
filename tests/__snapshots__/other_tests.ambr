# serializer version: 1
# name: test_smaller_than_five[1-2]
  True
# ---
# name: test_smaller_than_five[3-1]
  True
# ---
# name: test_smaller_than_five[4-2]
  False
# ---
# name: test_smaller_than_five[5-3]
  False
# ---
